scene/3d/label_3d.h

```diff
@@ -182,6 +185,16 @@ class Label3D : public GeometryInstance3D {
 	void set_text_direction(TextServer::Direction p_text_direction);
 	TextServer::Direction get_text_direction() const;
 
+	void set_localize_numeral_system(bool p_enable);
+	bool is_localizing_numeral_system() const;
+
+	void set_auto_translate(bool p_enable);
+	bool is_auto_translating() const;
+
+	_FORCE_INLINE_ String atr(const String p_string) const {
+		return is_auto_translating() ? tr(p_string) : p_string;
+	};
```

Comment:
I know it was copied from `control.h` as is but it's redundant there too.
```suggestion
	}
```